.gitignore

```diff
@@ -29,6 +29,7 @@ config/config.js
 lyftPrivateMethods.js
 uberPrivateMethods.js
 uberPrivateAPIresponses.js
+anotherConfig.js
```

Comment:
we should probably rename this to twilioConfig or something at some point, or just consolidate into one api key config file, but for now i think its fine